### DISCUSSION TITLE
Truncate does not work expected

### DISCUSSION THREAD

**Discussion by chalist** [tailwindlabs/tailwindcss#7211] [Help]
Look at this, please: https://codepen.io/chalist/pen/XWzWGma

Can I use truncate for `h3` tag?

**Comment by lukewarlow** [comment]
What's unexpected about this? It seems to work as expected to me?

![image](https://user-images.githubusercontent.com/32498324/151150744-041a5856-142c-4dab-a1a9-00cd02520e13.png)

**Reply by chalist** [reply]
Thanks for the reply. My container box has the `w-1/2` class. When I set `truncate` class, I see this:

![image](https://user-images.githubusercontent.com/1072154/151316712-0f8f48ff-120b-4641-bfbe-b561d22db534.png)

**Reply by lukewarlow** [reply]
Which browser is that?

**Reply by chalist** [reply]
Chromium
